modules/widgets/internet-defense-league.php

```diff
@@ -0,0 +1,154 @@
+<?php /*
+
+**************************************************************************
+
+Plugin Name:  Internet Defense League
+Description:  Displays your support for the Internet Defense League.
+Author:       Automattic Inc.
+Author URI:   https://automattic.com/
+
+**************************************************************************/
+
+class Internet_Defense_League_Widget extends WP_Widget {
```

Comment:
Maybe we could prefix it with `Jetpack_` to avoid any potential conflicts with other plugins or themes that may use the same class name?